securedrop/tests/test_journalist.py

```diff
@@ -559,6 +556,41 @@ def test_admin_add_user_too_short_username(self):
                                           is_admin=False))
         self.assertIn('Invalid username', resp.data)
 
+    def test_admin_sets_user_to_admin(self):
+        self._login_admin()
+        new_user = 'admin-set-user-to-admin-test'
+        resp = self.client.post(url_for('admin_add_user'),
+                                data=dict(username=new_user,
+                                          password=VALID_PASSWORD,
+                                          is_admin=False))
+        assert resp.status_code in (200, 302)
+        journo = Journalist.query.filter(Journalist.username == new_user).one()
+
+        resp = self.client.post(url_for('admin_edit_user', user_id=journo.id),
+                                data=dict(is_admin=True))
+        assert resp.status_code in (200, 302), resp.data.decode('utf-8')
+
+        # there are better ways to do this, but flake8 complains
+        journo = Journalist.query.filter(Journalist.username == new_user).one()
+        assert journo.is_admin is True
+
+    def test_admin_renames_user(self):
+        self._login_admin()
+        new_user = 'admin-renames-user-test'
+        resp = self.client.post(url_for('admin_add_user'),
+                                data=dict(username=new_user,
+                                          password=VALID_PASSWORD,
+                                          is_admin=False))
+        assert resp.status_code in (200, 302)
+        journo = Journalist.query.filter(Journalist.username == new_user).one()
+
+        new_user = new_user + 'a'
+        resp = self.client.post(url_for('admin_edit_user', user_id=journo.id),
+                                data=dict(username=new_user))
+        assert resp.status_code in (200, 302), resp.data.decode('utf-8')
+
```

Comment:
A comment like
<pre>
# the following will throw an exception if new_user is not found, therefore asserting it has been created
</pre>